src/core/Form/TextInput/TextInput.md

```diff
@@ -66,6 +67,23 @@ const status = errorState ? 'error' : 'default';
 </>;
 ```
 
+```js
+import { TextInput, Button } from 'suomifi-ui-components';
+<>
+  <TextInput
+    labelText="Test TextInput with fixed custom width of 250px"
+    status={status}
+    inputContainerProps={{ style: { width: '250px' } }}
+  />
+
+  <TextInput
+    labelText="Test TextInput with 100% width"
+    status={status}
```

Comment:
This seems unnecessary, since the example doesn't use status.

Reply:
True, was left there by accident. Should be now removed.